addons/stock/models/res_config_settings.py

```diff
@@ -48,6 +48,7 @@ class ResConfigSettings(models.TransientModel):
     annual_inventory_month = fields.Selection(related='company_id.annual_inventory_month', readonly=False)
     annual_inventory_day = fields.Integer(related='company_id.annual_inventory_day', readonly=False)
     group_stock_reception_report = fields.Boolean("Reception Report", implied_group='stock.group_reception_report')
+    module_stock_dropshipping = fields.Boolean("Dropshipping")
```

Comment:
Installed dropship from stock, created a dropship and ended in this state. It's not a great usalibility

![Odoo-DS-00001](https://user-images.githubusercontent.com/5512395/236506431-58fc6484-d079-4753-9a0d-db5780765294.png)

Reply:
waiting for THD's reply in the task once done will push all changes.

Reply:
Done